setup-cluster.sh

```diff
@@ -26,23 +39,23 @@ cat << EOF | tee ${VAGRANT_CONFIG}
 \$node_count = ${NODE_COUNT}
 \$master_count = 1
 
-\$vm_memory = 1024
-\$vm_vcpus = 1
-\$vm_disk = 45
+\$vm_memory = 16192
+\$vm_vcpus = 2
+\$vm_disk = 60
 \$node_box = "${NODE_BOX}"
 \$master_box = "${MASTER_BOX}"
 \$subnet = "192.168.156"
 EOF
 }
 
 declare -a CACHED_IMAGES=(
-    "$CACHE_IMAGE_PATH/1.5.0-rc"
+    "$CACHE_IMAGE_PATH/$VERSION"
```

Comment:
Great improvement on the logic. Instead of a static list we'll look for what the user asks 👍 . The only thing though, line 43 should not be loop since only 1 item will ever exist in this list. ```CACHED_IMAGES``` should really just be ```CACHED_IMAGE``` and the ```check-cached-images``` function should check to see if a file exists at the ```CACHED_IMAGE``` path and error if there's nothing.